CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/Panels` to show close button properly for night mode
 - `agate/LabeledIconButton` max-width so that huge sized icon will not be cut off
 - `agate/Spinner` to pause the animation when `paused` prop is true
+- `agate/Switch` icon position for all skins RTL locale and Electro/Titanium both LTR and RTL locale
```

Comment:
It might be better to replace `Switch` with `SwitchItem` because the `Switch` is a private component, app developers don't know it and they might use a `SwitchItem`.